main.py

```diff
@@ -45,10 +45,35 @@ def bna_pipeline():
             print(e)
             session.rollback()
 
-if __name__ == '__main__':
-    bna_pipeline()
+def ocm_pipeline():
+    # Read excel file as pandas dataframe
+    df = pd.read_csv(r'data/ocm_stations.csv')
+
+    # df_mapped = map_address(df_dropped)
+    engine = create_engine(db_uri, echo=True)
+    Session = sessionmaker(bind=engine)
+    session = Session()
+
+    for index, row in tqdm(df.iterrows()):
+        mapped_station = map_stations_ocm(row)
+        session.add(mapped_station)
+        session.flush()
```

Comment:
a short code comment why the flush is needed here, would help future developers

Reply:
we are doing the flush here to have the station id in our hand and use it for the address and charging